.github/workflow-template/jobs/e2e-risedev.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         timeout-minutes: 5
         run: |
           ~/cargo-make/makers ci-start ci-3cn-1fe
-          sqllogictest -p 4566 './e2e_test/streaming/**/*.slt'
+          sqllogictest -p 4566 -d dev './e2e_test/streaming/**/*.slt'
```

Comment:
If it will default to dev when it's not specified, I prefer to keep it what it was

```
sqllogictest -p 4566 './e2e_test/streaming/**/*.slt'
```

Reply:
Yes, the default user name is Unix user name, so we need to use dev.